automation/src/test/scala/org/broadinstitute/dsde/firecloud/page/workspaces/WorkspacePage.scala

```diff
@@ -36,28 +36,45 @@ abstract class WorkspacePage(namespace: String, name: String)(implicit webDriver
     (namespace, name) == readWorkspace
   }
 
+  def clickTab(tabName: String, pageUrl: String): Unit = {
+    try {
+      tabs.goToTab(tabName)
+    } catch {
+      case e: WebDriverException =>
+    }
+    // determine whether to retry click by comparing url
+    if (pageUrl != webDriver.getCurrentUrl) {
+      tabs.goToTab(tabName)
+    }
+  }
```

Comment:
Is this actually a problem that we've had?

Reply:
yes. if you trace this stack trace, tab click had no effect, causes one DataSpec test to fail.
https://fc-jenkins.dsp-techops.broadinstitute.org/job/firecloud-ui-fiab-test-runner/1711/testReport/org.broadinstitute.dsde.firecloud.test.metadata/DataSpec/Writer_and_reader_should_see_new_columns_with_no_defaults_or_local_preferences_when_analysis_run_that_creates_new_columns/

Reply:
Maybe the scroll problem again? Make `tab` in `TabBar` return a `Link` and `doClick()` it.

Reply:
this just happened again. MethodConfigSpec - looking for submissionId on wrong page. Monitor tab click is no-op.
https://fc-jenkins.dsp-techops.broadinstitute.org/job/swatomation-pipeline/4103/

Reply:
I really don't like this approach. Does fixing `tabs.goToTab` fix this instead?

Reply:
I'm not sure if `goToTab` also fixed this issue. I'm inclined to keep this because it's an extra insurance.

Reply:
I don't like the idea of insurance. If something is broken I want to know.

Reply:
instead of retry click, log a warning so that I know this is where test might have gone wrong.